test/Microsoft.AspNetCore.SignalR.Redis.Tests/SkipIfDockerNotPresentAttribute.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Testing.xunit;
+
+namespace Microsoft.AspNetCore.SignalR.Redis.Tests
+{
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
```

Comment:
Is it possible to make this attribute work on a class as well - I think this would be more useful

Reply:
Unfortunately it doesn't seem like xunit has support for that